terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -1100,7 +1150,8 @@ def remove_inconsequential_statements(statements: List[JSON]) -> List[JSON]:
                                --network gitlab-runner-net \
                                --env DOCKER_HOST=tcp://gitlab-dind:2375 \
                                gitlab/gitlab-runner:v13.10.0
-                    """[1:]),  # trim newline char at the beginning as dedent() only removes indent common to all lines
+                    """[1:]),  # Trim newline char at the beginning as dedent() only removes indent
+                # common to all lines
```

Comment:
alignment?

Reply:
`make format` forces the indentation like this. The only other option is to move the comment to the line after, which is a little odd, but I'll try and make it clear.